Dockerfile.centos.patch

```diff
@@ -0,0 +1,27 @@
+--- Dockerfile	2020-10-16 10:26:19.124105649 -0400
++++ Dockerfile.centos	2020-10-16 10:28:25.713003840 -0400
+@@ -1,4 +1,4 @@
+-FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
++FROM registry.svc.ci.openshift.org/openshift/release:golang-1.14 AS builder
+ WORKDIR /go/src/github.com/openshift/cluster-logging-operator
+ 
+ # COPY steps are in the reverse order of frequency of change
+@@ -17,9 +17,8 @@
+ 
+ RUN make build
+ 
+-FROM registry.svc.ci.openshift.org/ocp/4.7:cli AS origincli
++FROM centos:centos7 AS centos
+ 
+-FROM registry.svc.ci.openshift.org/ocp/4.7:base
+ RUN INSTALL_PKGS=" \
+       openssl \
+       rsync \
+@@ -38,7 +37,6 @@
+ COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/manifests /manifests
+ RUN rm /manifests/art.yaml
+ 
+-COPY --from=origincli /usr/bin/oc /usr/bin/
```

Comment:
this is not the same based on the change:
* former provided `oc`
* later provides must-gather scripts

Would not we need both?  Maybe not needed if we are only doing devel

Reply:
I suggest we may need a compromise here, because we won't get both Centos-based development and oc. We should clearly state for external devs that centos based development has the limitation of not being able to using the final image for must-gather executions

Reply:
OC is not available without the origincli images, which are not public. However our own makefile will download it on demand so I think this works OK for developers - it's never going to be used for CI.